lib/xregexp.js

```diff
@@ -4,7 +4,7 @@ const originalXRegExp = require('xregexp');
  * Wrapper for xRegExp to work around
  * https://github.com/slevithan/xregexp/issues/130
  */
-function xRegExp(regexp, flags) {
+export default function xRegExp(regexp, flags) {
```

Comment:
Unrelated, but I think we should be able to update xregexp and delete this wrapper file.